app/routes/orders/new.js

```diff
@@ -35,7 +35,8 @@ export default Route.extend({
       form  : await eventDetails.query('customForms', {
         'page[size]' : 50,
         sort         : 'id'
-      })
+      }),
+      settings: await this.store.queryRecord('setting', {})
```

Comment:
@kushthedude why is this needed ? We already have the settings service globally injected into all components/controllers/routes ... You can directly use `settings` from within the component without passing it to it.

Reply:
@niranjan94 , when I tried to use injected settings , it required a refresh to change the value to latest updated expiry time which needed to define a new action refreshRoute, thats why I injected settings again and it is working fine now

Reply:
@kushthedude then you'll have to make changes to settings service to ensure it stays updated on changes. As far as settings are concerned, the settings service should be the single point of truth. 

Please refrain from falling back to shortcuts instead of resolving the actual problem.